components/raftstore/src/coprocessor/split_check/mod.rs

```diff
@@ -128,3 +128,18 @@ impl<'a, E> Host<'a, E> {
         self.cfg.region_bucket_size.0
     }
 }
+
+#[inline]
+pub fn calc_split_keys_count(
+    count_per_region: u64,
+    split_threshold: u64,
+    batch_split_limit: u64,
+) -> u64 {
+    let actual_split_limit = if count_per_region % split_threshold < split_threshold / 2 {
```

Comment:
I think it should be `cmp::min(count_per_region.saturating_sub(split_threshold / 2) / split_threshold, batch_split_limit)`.

Reply:
done